source/opt/cfg_cleanup_pass.cpp

```diff
@@ -127,8 +111,8 @@ void CFGCleanupPass::RemovePhiOperands(
       // means that this |phi| argument is no longer defined. Replace it with
       // |undef_id|.
       if (!undef_id) {
-        type_id = def_use_mgr_->GetDef(arg_id)->type_id();
-        undef_id = TypeToUndef(type_id);
+        type_id = get_def_use_mgr()->GetDef(arg_id)->type_id();
+        undef_id = Type2Undef(type_id);
```

Comment:
I prefer "To" to "2" in these situations.
"Nothing compares 2 u" is great for a song, but not for a method name. :-)

This is not a request to change.    :-)